base_classes/NXroot.nxdl.xml

```diff
@@ -1,10 +1,10 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl" ?>
+<?xml version='1.0' encoding='UTF-8'?>
+<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl"?>
 <!--
 # NeXus - Neutron and X-ray Common Data Format
-#
-# Copyright (C) 2008-2022 NeXus International Advisory Committee (NIAC)
-#
+# 
+# Copyright (C) 2014-2023 NeXus International Advisory Committee (NIAC)
```

Comment:
As stated [elsewhere](https://github.com/nexusformat/definitions/pull/1303#issuecomment-1806853868), do not change the copyright dates